Extract more output (#36)
USGS-R/lake-temperature-process-models: README.md

-----------------
## Dependent files from [`lake-temperature-model-prep pipeline`](https://github.com/USGS-R/lake-temperature-model-prep)
_Files that will eventually be transferred using GLOBUS:_
  * Lake - GCM cell tile crosswalk: `'1_prep/in/lake_cell_tile_xwalk.csv'`

>     * Created within [the `targets` sub-pipeline](https://github.com/USGS-R/lake-temperature-model-prep/blob/main/_targets.R#L194-L213) of the repo

Reviewer: Linking to a specific line in the repo in your documentation is a bit risky because the code could change and break your reference, especially if we add/adjust targets above that one linked. Perhaps this would be more robust to future changes?

```suggestion
    * Created within [the `targets` sub-pipeline](https://github.com/USGS-R/lake-temperature-model-prep/blob/main/_targets.R), look for the `lake_cell_tile_xwalk_df` target
```
